proposals/wg-image-compatibility.md

```diff
@@ -0,0 +1,104 @@
+# OCI Working Group Proposal: Image Compatibility
+
+Proposal created from [OCI WG template](https://github.com/opencontainers/tob/blob/master/WG-TEMPLATE.md).
+
+## Image Compatibility OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the Image Compatibility Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”).
+The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter.
+Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+In mission-critical industries, applications often require special features provided by host operating systems.
+These are usually described in the release notes or installation guides.
+We believe we have identified the missing puzzle in the OCI standard, the image compatibility specification, which is a key feature that describes the requirements of special containerized applications that have hard requirements for specific kernel versions, configurations, modules, or out-of-tree drivers.
```

Comment:
> the image compatibility specification

whether to extending the image compatibility capability in platform object, new fields, annotations, or in artifact, could be investigated by the working group based on various use cases/requirements. 
therefore it would be better to use "image compatibility" instead.

Reply:
Fixed.